tests/e2e/test_fuzzing.py

```diff
@@ -41,3 +41,24 @@ def test_fuzzing(protostar, copy_fixture):
 """
         in result
     )
+
+
+@pytest.mark.usefixtures("init")
+def test_fuzzing_changing_seed(protostar, copy_fixture):
+    seeds = []
+    copy_fixture("test_fuzz_single.cairo", "./tests")
+
+    for i in range(0, 5):
```

Comment:
I'm not sure if this number is good, the tests are pretty slow (in my local env, one test takes ~6s, so here we'll have 30s)

Reply:
Running tests twice should be enough.